Update ko.lang

DDinghoya opened the pull request.

SuperrSonic commented:
The art viewer line has an incomplete msgid. I can add it myself but won't be able to for a while.

Could you also add "Art Folder" too? It's the latest setting.

DDinghoya commented:
> The art viewer line has an incomplete msgid. I can add it myself but won't be able to for a while.
> 
> Could you also add "Art Folder" too? It's the latest setting.

Yup! Changed type error of  art viewer line. And added "Art Folder" line.
